packages/checkout/widgets-lib/src/widgets/sale/hooks/useSaleEvents.ts

```diff
@@ -27,6 +27,7 @@ export const useSaleEvent = () => {
     recipientAddress: userId,
     signResponse,
     paymentMethod,
+    orderQuote,
```

Comment:
orderQuote has too much information is not needed, can we change this to `totalAmount`.

ie: config.contractId should not be sent, as well as `products` is info that gets passed again towards the end on the success event